cli/tsconfig.components.json

```diff
@@ -1,10 +1,12 @@
 {
     "extends": "../tsconfig.json",
     "compilerOptions": {
-        "target": "es5",
+        "allowJs": false,
```

Comment:
We don't allow JavaScript, because 
1. We compile from `/.components` into the same directory (`/.components`). `tsc` can't compile `/.components/*.js` into `/.components/*.js`, because it can't override the source files it's compiling
2. We change the extension of all the JavaScript files to TypeScript while copying them into `/.components`, so we won't have JavaScript files inside `/.components`